docs/interactions/Application_Commands.md

```diff
@@ -10,20 +12,24 @@ Application commands are commands that an application can register to Discord. T
+| guild_id?                  | snowflake                                                                                                                                      | guild id of the command, if not global                                                                                                                                      | all         |
+| name                        | string                                                                                                                                         | [Name of command]((#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-naming)), 1-32 characters                                | all         |
+| name_localizations?         | ?dictionary with keys in [available locales](#DOCS_REFERENCE/locales)                                                                          | Localization dictionary for `name` field. Values follow the same restrictions as `name`                                                                            | all         |
+| description                 | string                                                                                                                                         | Description for `CHAT_INPUT` commands, 1-100 characters. Empty string for `USER` and `MESSAGE` commands                                                            | all         |
+| description_localizations?  | ?dictionary with keys in [available locales](#DOCS_REFERENCE/locales)                                                                          | Localization dictionary for `description` field. Values follow the same restrictions as `description`                                                              | all         |
+| options?                    | array of [application command option](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-option-structure) | Parameters for the command, max of 25                                                                                                                              | CHAT_INPUT  |
+| default_member_permissions? | string                                                                                                                                         | Set of [permissions](#DOCS_TOPICS_PERMISSIONS) represented as a bit set                                                                                            | all         |
+| dm_permission?              | boolean                                                                                                                                        | Indicates whether the command is available in DMs with the app, only for globally-scoped commands                                                                  | all         |
```

Comment:
```suggestion
| dm_permission?              | ?boolean                                                                                                                                        | Indicates whether the command is available in DMs with the app, only for globally-scoped commands                                                                  | all         |
```

This is null when not set (should it be?)

Reply:
🤦 did the wrong one, thanks for getting the right one lol

Reply:
Hmmm gonna check on this with engineering before updating

Reply:
I meant default_member_permissions which you  did

Reply:
Also, the danger box below implies that `dm_permission` defaults to `false`. It may be useful to also point out this in the field's description.

Reply:
it defaults to default_permission

Reply:
Checked with eng team, and we're going to update it to default to `true` instead of it being nullable. Updated the docs accordingly

Reply:
awesome thanks!

Reply:
hmmm actually im going to revert back and make it nullable for now just to make sure it's consistent with its current state. i'll work on getting the change pushed through in the next couple of days and update the docs then.

thanks for pointing this all out in the first place @ckohen

Reply:
the UI's behavior currently acts as true though (if it's not passed, it shows up in DMs), which I'll also include